examples/extensions/logger.middleware.ts

```diff
@@ -0,0 +1,55 @@
+import { Service } from "typedi";
+import { GraphQLResolveInfo, GraphQLFieldConfig, GraphQLObjectTypeConfig } from "graphql";
+
+import { MiddlewareInterface, NextFn, ResolverData } from "../../src";
+
+import { Context } from "./context.interface";
+import { Logger } from "./logger.service";
+
+const extractFieldConfig = (info: GraphQLResolveInfo): GraphQLFieldConfig<any, any> => {
```

Comment:
Move `extractFieldConfig` and `extractParentConfig` into helpers - they are a generic `info` parse helpers

Reply:
You mean to extract them to `src/helpers` correct? If so, I have done it here: 1779546e84b9ba1085ea81503fb280f781d87b7c

I have also added some basic tests for them as I think it's more proper if they are part of the actual source code and not just examples.

I have also renamed `extractParentConfig` -> `extractParentTypeConfig`, seemed more appropriate when using it, but I can change it back of course if you prefer.

Reply:
> You mean to extract them to src/helpers correct?

No, into example helpers, to not pollute the log middleware code

Reply:
Ah, alright my mistake, have relocated them into example helpers in this commit then: 19b68eece62205dc6e97b69d4bebc898a6e7a3ad